tests/helpers/healthevent.go

```diff
@@ -161,15 +161,15 @@ func (h *HealthEventTemplate) WriteToTempFile() (string, error) {
 	return tempFile.Name(), nil
 }
 
-func SendHealthEventsToNodes(nodeNames []string, eventFilePath string) error {
+func SendHealthEventsToNodes(nodeNames []string, eventFilePath string, errorCode string, checkName string) error {
```

Comment:
Instead of adding more param, can we use the stablished pattern -- https://github.com/NVIDIA/NVSentinel/blob/85c726b6cb8009424e63f73d938dc5e808bf99d9/tests/scale_test.go#L207-L211